cookbook-development.gemspec

```diff
@@ -18,18 +18,18 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'berkshelf', '~>3.1'
-  spec.add_dependency 'bundler', '~> 1.3'
-  spec.add_dependency 'chef', '~> 11.6'
+  spec.add_dependency 'berkshelf', '~>3.3'
+  spec.add_dependency 'bundler', '~> 1.10'
+  spec.add_dependency 'chef', '~> 11.18'
```

Comment:
Yeah - was wondering about that - as long as we are cool risking breaking all the things if there are cookbooks out there that haven't been updated...